R/Simulations-class.R

```diff
@@ -81,17 +86,17 @@
         o$result()
       }
   )
-validObject(.GeneralSimulations())
 
-##' Initialization function for "GeneralSimulations"
-##'
-##' @param data see \code{\linkS4class{GeneralSimulations}}
-##' @param doses see \code{\linkS4class{GeneralSimulations}}
-##' @param seed see \code{\linkS4class{GeneralSimulations}}
-##' @return the \code{\linkS4class{GeneralSimulations}} object
-##'
-##' @export
-##' @keywords methods
+## constructor ----
+
+#' @rdname GeneralSimulations-class
+#'
+#' @param data (`list`)\cr see slot definition.
+#' @param doses (`numeric`)\cr see slot definition.
+#' @param seed (`integer`)\cr see slot definition.
+#'
+#' @example examples/Simulations-class-GeneralSimulations.R
+#' @export
```

Comment:
please move the line `# nolint start` from the top of the file to after the code you cleaned up, meaning that linter will apply to the cleaned up area